frontend/src/components/Toast.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect } from "react";
+import xgreen from "../icons/x-green.svg";
+
+export const Toast = ({ onCloseToast }: { onCloseToast: () => void }) => {
+  useEffect(() => {
+    setTimeout(() => {
+      onCloseToast();
+    }, 3000);
+  });
+  return (
+    <div className="toast-box" role="status">
+      <p className="toast-message">All changes saved!</p>
+      <button onClick={onCloseToast} className="toast-button">
```

Comment:
I see the old message to the right and the new one to the left. Its confusing.

Reply:
I added a commit to make the "Unsaved changes" warning disappear as soon as the saving process starts. It will re-appear in case saving wasn't successful.